cli/src/config-schema.json

```diff
@@ -334,6 +334,11 @@
                     "type": "string",
                     "description": "Revisions to give shorter change and commit IDs to",
                     "default": "<revsets.log>"
+                },
+                "always-allow-large-revsets": {
+                    "type": "boolean",
+                    "description": "Whether to allow large revsets to be used in all commands without the `all:` modifier",
```

Comment:
Nit: I think large is the wrong word for it, maybe `non-singleton-revsets`?

Reply:
FWIW, the name "large" comes from the fact that before the `all:` prefix existed (last year), you used the `--allow-large-revsets`/`-L` flag to `rebase` instead. See 9c8d8b73. So, it's really just bringing that name back from the grave. I think `singleton` is a bit too wordy, tbh.

Reply:
Makes sense, and I agree it's wordy/technical. We can bikeshed the wording some other time when we come up with a better term